internal/test/issues/issue-1039/issue_test.go

```diff
@@ -0,0 +1,282 @@
+				assert.Falsef(t, obj.SearchBox.IsNull(), "search box should not be null")
+
+				assert.Falsef(t, obj.Popularity.IsSpecified(), "popularity should not be set")
+				assert.Falsef(t, obj.Popularity.IsNull(), "popularity should not be null")
+
+				assert.Falsef(t, obj.BreakoutGroup.IsSpecified(), "breakout group should not be set")
+				assert.Falsef(t, obj.BreakoutGroup.IsNull(), "breakout group should not be null")
+
+				// check for non-nullable field
+				assert.Truef(t, obj.HomeCountry == nil, "home country should be nil")
+			},
+		},
+
+		{
+			name: "when only organization name is provided and alias name is null ",
+			json: []byte(`{"organization":{"name":"test-name","alias_name":null}}`),
+			assert: func(t *testing.T, obj PatchRequest) {
+				t.Helper()
+				// check for nullable field
+				assert.Truef(t, obj.Organization.IsSpecified(), "organization should be set")
+				assert.Falsef(t, obj.Organization.IsNull(), "organization should not be null")
+
+				gotOrg, err := obj.Organization.Get()
+				require.NoError(t, err)
+				assert.Equalf(t, "test-name", string(*gotOrg.Name), "name should  be test-org")
+
+				assert.Truef(t, gotOrg.AliasName.IsSpecified(), "alias name should be set")
+				assert.Truef(t, gotOrg.AliasName.IsNull(), "alias name should be null")
+
+				assert.Falsef(t, obj.SearchBox.IsSpecified(), "search box should not be set")
+				assert.Falsef(t, obj.SearchBox.IsNull(), "search box should not be null")
+
+				assert.Falsef(t, obj.Popularity.IsSpecified(), "popularity should not be set")
+				assert.Falsef(t, obj.Popularity.IsNull(), "popularity should not be null")
+
+				assert.Falsef(t, obj.BreakoutGroup.IsSpecified(), "breakout group should not be set")
+				assert.Falsef(t, obj.BreakoutGroup.IsNull(), "breakout group should not be null")
+
+				// check for non-nullable field
+				assert.Truef(t, obj.HomeCountry == nil, "home country should be nil")
+			},
+		},
+
+		{
+			name: "when search box is null ",
+			json: []byte(`{"search_box":null}`),
+			assert: func(t *testing.T, obj PatchRequest) {
+				t.Helper()
+				// check for nullable field
+				// check for nullable fields
+				assert.Falsef(t, obj.Organization.IsSpecified(), "organization should not be set")
+				assert.Falsef(t, obj.Organization.IsNull(), "organization should not be null")
+
+				assert.Truef(t, obj.SearchBox.IsSpecified(), "search box should be set")
+				assert.Truef(t, obj.SearchBox.IsNull(), "search box should be null")
+
+				assert.Falsef(t, obj.Popularity.IsSpecified(), "popularity should not be set")
+				assert.Falsef(t, obj.Popularity.IsNull(), "popularity should not be null")
+
+				assert.Falsef(t, obj.BreakoutGroup.IsSpecified(), "breakout group should not be set")
+				assert.Falsef(t, obj.BreakoutGroup.IsNull(), "breakout group should not be null")
+			},
+		},
+
+		{
+			name: "when search box is null and organization has non zero value",
```

Comment:
I have checking that to ensure that because the other fields are not provided -- it should be basically not set and not null. But I think now when I am reading your comment, probably, we do not need to duplicate this in every test case.